Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -18,6 +18,14 @@ block FirstOrderAMIGO
   parameter Real r(
     min=100*Buildings.Controls.OBC.CDL.Constants.eps)=1
     "Typical range of control error, used for scaling the control error";
+  parameter Real yHig = 1
+    "Higher value for the relay output";
+  parameter Real yLow = 0.1
+    "Lower value for the relay output";
+  parameter Real deaBan = 0.1
+    "Deadband for holding the output value";
```

Comment:
Could you please give more instruction of how to set the `yHig`, `yLow`, `deaBan` and `yRef` value? I see in the example `Buildings.Fluid.HeatExchangers.Examples.WetCoilCounterFlowPControlAutoTuning`, you set different values, especially the `deaBan=1/298.15` and `yRef=0.5`.

Could these parameters be any value, as long as satisfying the assert condition?

Reply:
The reason to change<code>yHig</code>, <code>yLow</code>, and <code>yRef</code> is to avoid <i>log(0)</i> for the following equation:

![image](https://github.com/lbl-srg/modelica-buildings/assets/6975889/0283e440-e799-4234-85f6-c3ca8df14676)

I believe <code>deaBan</code> is more critical for the performance and I added one sentence to provide a general suggestion.
Please do let me know if it is appropriate.